docs/TESTING.md

```diff
@@ -0,0 +1,55 @@
+# Testing
+
+Testing is an integral part of creating new features and maintaining the software.
+
+## Automated testing
+
+A [few actions](https://github.com/Automattic/vip-design-system/tree/trunk/.github/workflows) are automatically run via Github Actions when a pull request is created or updated.
+
+### Linting
+
+We run the following checks:
+
+- linting (`npm run lint`)
+- format checking (`npm run format:check`)
+- type checks (`npm run check-types`)
+
+### Unit tests
+
+Located in [test/](https://github.com/Automattic/vip-design-system/tree/trunk/test).
+
```

Comment:
Recommend adding sections on `dependaban` and `build` as well for completeness (see [ci.yml](https://github.com/Automattic/vip-design-system/blob/trunk/.github/workflows/ci.yml))